checkbox-ng/debian/changelog

```diff
@@ -1,3 +1,9 @@
+checkbox-ng (2.2.0) unstable; urgency=medium
+
+  * "new upstream version"
+
+ -- Devices Certification Bot <robot@canonical.com>  Thu, 23 Feb 2023 04:21:31 +0000
+
```

Comment:
Is it expected that we start the Debian changelog for the next version now?

Reply:
You're right, this pre-versionning always assumes a minor+1 increment. Let's bump deb and snaps versions only when ready to initiate a release. Moreover with weekly releases, the version pattern will be different.